creating_diverse_inclusive_communities.adoc

```diff
@@ -34,7 +36,11 @@ DrupalCon offers scholarships for attendees and speakers of minority status.
 Several mainstage keynotes at All Things Open 2019 discussed diversity.
 With so much attention from community leaders, why do contributor data still paint such a damning picture?
 
-Relative lack of open source activity from women and under-represented minorities (URMs) seems surprising at first, but a closer look reveals some persistent problems regarding inclusion in open source communities. Inclusion is often used as a synonym for diversity; in fact, it is its own principle. To quote Meg Bolger, "Inclusion is about folks with different identities feeling and/or being valued, leveraged, and welcomed within a given setting (e.g., your team, workplace, or industry)." There are some persistent, systemic barriers that keep women and underrepresented minorities (URMs) from being included in open source communities. Without addressing and rectifying these systemic problems, diversity in open source won't improve
+Relative lack of open source activity from women and under-represented minorities (URMs) seems surprising at first, but a closer look reveals some persistent problems regarding inclusion in open source communities.
+Inclusion is often used as a synonym for diversity; in fact, it is its own principle.
+To quote Meg Bolger, "Inclusion is about folks with different identities feeling and/or being valued, leveraged, and welcomed within a given setting (e.g., your team, workplace, or industry)."
```

Comment:
Should we provide title/affiliation/pronouns here, as you do for all other interviewees?

Reply:
Good question. I would say no because I didn't interview Meg directly as a primary source, but if you think it looks out of place, I don't oppose it.

Reply:
Good question. I'm tempted to say no because Meg isn't a primary source whom I did an original interview with for this chapter. But if you feel it looks out of place, I'm not opposed to it.

Reply:
I think the phrase "to quote" along with the footnote to the source is clear indication it is not a primary source, and as long as we are consistent it's OK with me to leave those items out (as the secondary source covers that effectively within the cited work itself).

Reply:
Great! All sounds good to me. I just wanted to double check.